shared/structures/src/members/records/RecordSettings.ts

```diff
@@ -242,6 +243,9 @@ export class RecordSettings extends AutoEncoder {
     @field({ decoder: new ArrayDecoder(ResolutionRequest), optional: true })
     resolutions?: ResolutionRequest[];
 
+    @field({ decoder: new EnumDecoder(PermissionLevel), ...NextVersion })
+    userManagerPermissionLevel = PermissionLevel.Write;
```

Comment:
Misschien moeten we het vanaf nu overal member manager noemen in plaats van user manager, om verwarring te vermijden

Reply:
```suggestion
    memberPermissionLevel = PermissionLevel.Write;
```

Reply:
En omdat record settings ook buiten ledenadministratie gebruiken, misschien toch nog iets generieker kiezen

Reply:
```suggestion
    externalPermissionLevel = PermissionLevel.Write;
```
```suggestion
    userPermissionLevel = PermissionLevel.Write;
```
```suggestion
    memberPermissionLevel = PermissionLevel.Write;
```

🤷‍♂️